src/Esri.ArcGISRuntime.Toolkit/XamarinForms/Compass/Compass.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Xamarin.Forms;
+using Esri.ArcGISRuntime.Toolkit.Xamarin.Forms.Internal;
+using Esri.ArcGISRuntime.Xamarin.Forms;
+using System.ComponentModel;
+
+namespace Esri.ArcGISRuntime.Toolkit.Xamarin.Forms
+{
+    /// <summary>
+    /// The Compass Control showing the heading on the map when the rotation is not North up / 0.
+    /// </summary>
+    public class Compass : View
+    {
+        internal readonly UI.Controls.Compass NativeCompass;
+        private bool _headingSetByGeoView;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Compass"/> class
+        /// </summary>
+        public Compass() : this(new UI.Controls.Compass()) { }
+
+        internal Compass(UI.Controls.Compass nativeCompass)
+        {
+            NativeCompass = nativeCompass;
+
+#if NETFX_CORE
+            nativeCompass.SizeChanged += (o, e) => InvalidateMeasure();
+#endif
+
+            var tapRecognizer = new TapGestureRecognizer();
+            tapRecognizer.Command = new TapCommand(ResetRotation);
```

Comment:
Why use the command instead of the `Tapped` event?  If we need to use a command, maybe grab `DelegateCommand` from `LayerContentViewModel` ([here](https://github.com/Esri/arcgis-toolkit-dotnet/blob/ae0c2592d5bd54fec18e0a901841a93db0cd8582/src/Esri.ArcGISRuntime.Toolkit/Shared/UI/Controls/LayerList/LayerContentViewModel.cs#L402)) and use that instead?

Reply:
Not available on UAP: 
![image](https://user-images.githubusercontent.com/1378165/36493138-c019ae52-16e3-11e8-8f65-7a599261430f.png)